hare/eligibility/beacon.go

```diff
@@ -34,5 +37,22 @@ func (b *beacon) Value(layer types.LayerID) (uint32, error) {
 		return nilVal, err
 	}
 
-	return v, nil
+	return getBeaconFromSet(v), nil
+}
+
+func getBeaconFromSet(bids map[types.BlockID]struct{}) uint32 {
```

Comment:
Just rename to calcValue.
You can add a comment above the func dec:
// calculates the beacon value from the set of ids